docs/how-tos/4_schedule_a_script.md

```diff
@@ -6,11 +6,12 @@ title: Schedules
 # Run on a schedule
 
 Windmill allows you to define schedules for Scripts and Flows. Once a schedule
-is defined, it will automatically run the script at the set frequency. Think 
-of it as an easy-to-use cron scheduler that you can share with other users.
+is defined, it will automatically run the script at the set frequency. Think of
+it as an easy-to-use cron scheduler that you can share with other users.
 
-A schedule consists of a Script/Flow, its script arguments, and a CRON 
-expression that controls the execution frequency. 
+A schedule consists of a Script/Flow, its script arguments, and a CRON
+expression that controls the execution frequency (if you're confused with CRON
+syntax make sure to use https://crontab.guru)
```

Comment:
https://crontab.guru is really great for adjusting cron expression to the actual need